core/include/detray/navigation/navigator.hpp

```diff
@@ -729,7 +732,10 @@ class navigator {
         // If no trust could be restored for the current state, (local)
         // navigation might be exhausted: re-initialize volume
         else {
-            init(track, navigation, cfg, ctx);
+            // Use overstep tolerance instead of path tolerance
+            const bool use_path_tolerance = false;
```

Comment:
I feel like the naming is a bit confusing, because the path tolerance is always used in the init function. Is it possible to rename to e.g. ```check_overstepping```?

Reply:
It still does overstepping check. The path tolerance is used as overstep tolerance if it is true

Reply:
I will change use_path_tolerance_as_overstep_tolerance then

Reply:
> That tolerance is too small to catch overstepping 

The change is only for when entering new volume and starting the propagation, which is not really a concern.

> So it does not need to be checked again

You can open a new PR to ignore overstep checking - I don't know how to